apps/synt-vedtakshistorikk-service/src/main/java/no/nav/testnav/apps/syntvedtakshistorikkservice/service/ArenaForvalterService.java

```diff
@@ -166,6 +171,20 @@ private void opprettArbeidssoeker(
         }
     }
 
+    private void checkNyeBrukereResponse(NyeBrukereResponse nyeBrukereResponse, String personident) {
+        String feilmelding = null;
+        if (isNull(nyeBrukereResponse)) {
+            feilmelding =  String.format("Kunne ikke opprette ny bruker med fnr %s i Arena: %s", personident, "Ukjent feil.");
+        } else if (nonNull(nyeBrukereResponse.getNyBrukerFeilList()) && !nyeBrukereResponse.getNyBrukerFeilList().isEmpty()) {
+            feilmelding =  String.format("Kunne ikke opprette ny bruker med fnr %s i Arena: %s", personident, nyeBrukereResponse.getNyBrukerFeilList().get(0).getMelding());
+        }
+        if (feilmelding != null){
```

Comment:
Hvis du bruker isNull() bør du også benytte nonNull() for konsistens, men aldri på Strings, bruk 
StringUtils (fra apache.common biblioteket) og isBlank() eller isNotBlank() der begge sjekker både null og tomme stringer.